.gitignore

```diff
@@ -27,3 +27,6 @@
 
 # docs build
 /docs/build/
+
+# forecasts-ncov is a directory (git repo) introduced at build time
+forecasts-ncov/
```

Comment:
I'd rather not introduce whole new chunks of code at build time that are unrelated to the code in this repo, as it adds lots of unnecessary complexity during deployment which can be a rich source of bugs. For example, it allows changes in the forecasts-ncov repo to break our build (or deployed app) on Heroku when things worked fine locally or in CI. It also means that redeploys without any code changes on our end will "auto-upgrade" the forecasts-ncov viz app. Some of those are addressable by fetching a static ref instead or trying to be extra careful, but it seems better to me to make them non-issues altogether.

Instead, we can vendor the forecasts-ncov repo and then control in this repo when we take changes from it. This provides a much more controlled and reproducible build and deployment process. I've had good experiences over the years with `git subrepo` (a third-party tool) to do this, and there's now also `git subtree` in core git to accomplish similar ends.

Reply:
We can vendor it for sure. While this is a temporary solution, I understand we want it to be reproducible while it lasts. I see the medium term direction as something like this:
 
1. Turn the viz app into published, versioned importable react components (and probably this no longer lives in forecasts-ncov)
2. Remove Gatsby, and whatever we use to replace it then has forecasting pages which import those react components

Reply:
This was a good example to test out `git subtree` which I'd read about but not used - thanks!